utils/hlsFormats.ts

```diff
@@ -78,10 +74,6 @@ export function getHlsAudioCodecs(
     }
   }
 
-  if (hasMp3Support(videoTestElement)) {
-    hlsVideoAudioCodecs.push('mp3');
-  }
-
```

Comment:
Why?

Reply:
Shaka Player doesn't output any audio when MP3 is declared as supported.

This may be related: https://github.com/google/shaka-player/issues/2337

The server is also way too eager to transcode other formats to MP3, when AAC is also always supported and has better quality and lower file sizes.

Reply:
@MrTimscampi Great. I also don't see the reason to use MP3, in fact I think we should deprecate it altogether in our current web (although maybe not for songs). Just wanted to know the reasoning behind.